CHANGELOG.md

```diff
@@ -1,5 +1,435 @@
+  This allows Yew more flexibility when parsing in the `html!` macro which opens the door for some
+  new features too!
+  - [Rewrite of `agent` module to new `yew-agent` crate!](https://github.com/yewstack/yew/pull/1842).
+  - [Removal of `services` module](https://github.com/yewstack/yew/issues/1841).
+  
+    This removes the whole module from the project and the following are the recommended replacements: 
+    - `ConsoleService` - [weblog](https://crates.io/crates/weblog) or [gloo_console](https://crates.io/crates/gloo-console)
+    - `DialogService` - [gloo_dialogs](https://crates.io/crates/gloo-dialogs)
+    - `IntervalService` - [gloo-timers](https://crates.io/crates/gloo-timers)
+    - `KeyboardService` - `onkeydown` / `onkeypress` / `onkeyup` like so:
+      ```rust
+      let callback = Callback::from(|e| {
+        e.prevent_default();
+        todo!("use `e`, like you would in service methods.");
+      });
+
+      html! {
+        <input onkeydown={callback} />
+      }
+      ```
+    - `ResizeService` - use `EventListener` from [gloo_events](https://crates.io/crates/gloo-events)
+    to attach the event listener instead.
+    - `StorageService` - [gloo-storage](https://crates.io/crates/gloo-storage)
+    - `TimeoutService` - [gloo-timers](https://crates.io/crates/gloo-timers)
+    - `WebSocketService` - [wasm-sockets](https://crates.io/crates/wasm-sockets) or
+    [reqwasm](https://crates.io/crates/reqwasm)
+    - `FetchService` - [reqwest](https://crates.io/crates/reqwest) or [reqwasm](https://crates.io/crates/reqwasm)
+  
+  - [Removal of the `format` module](https://github.com/yewstack/yew/pull/1842).
+
+    Removal of `services` made `format` redundant and could cause conflicts or restrictions when
+    using other libraries.  
+
+  - [New Yew app entry points, multiple Yew apps enabled with `AppHandle` to control Yew app lifetime](https://github.com/yewstack/yew/pull/1825).
+  
+    This also removes the `App` struct, check out the new example `dyn_create_destroy_apps` to see
+    these changes in action!
+
+    Yew app entry function changes:
+    - `start_app<COMP>()` => `start_app<COMP>() -> AppHandle<COMP>`
+    - `start_app_with_props<COMP>(COMP::Properties)` => `start_app_with_props<COMP>(COMP::Properties) -> AppHandle<COMP>`
+    - `App::<COMP>::mount(self, Element) -> ComponentLink<COMP>` => `start_app_in_element<COMP>(Element) -> AppHandle<COMP>`
+    - `App::<COMP>::mount_with_props(self, Element, COMP::Properties) -> ComponentLink<COMP>` => `start_app_with_props_in_element<COMP>(Element, COMP::Properties) -> AppHandle<COMP>`
+    - `App::<COMP>::mount_as_body(self) -> ComponentLink<COMP>` => `start_app_with_props_in_element<COMP>(Element, COMP::Properties) -> AppHandle<COMP>`
+    - `App::<COMP>::mount_as_body_with_props(self, COMP::Properties) -> ComponentLink<COMP>` => `start_app_with_props_as_body<COMP>(COMP::Properties) -> AppHandle<COMP>`
```

Comment:
Not happy how this gets displayed 😞
I don't know whether one after the other is better in a rust block:
```rust
// v0.18.0
fn start_app<COMP: Component>();
// v0.19.0
fn start_app<COMP: Component>() -> AppHandle<COMP>;
// etc
```